scripts/local-universe.py

```diff
@@ -115,38 +171,121 @@ def enumerate_docker_images(package_path):
     return (name for _, name in dockers.items())
 
 
-def download_docker_image(name, use_sudo):
+@contextlib.contextmanager
+def run_docker_registry(volume_path):
```

Comment:
For my own edification why are we running the docker registry and kill it after we are done?